src/store.c

```diff
@@ -173,18 +233,20 @@ struct rill_store *rill_store_open(const char *file)
         goto fail_mmap;
     }
 
-    store->head = store->vma;
-    store->vals = (void *) ((uintptr_t) store->vma + store->head->vals_off);
-    store->data = (void *) ((uintptr_t) store->vma + store->head->data_off);
-    store->index = (void *) ((uintptr_t) store->vma + store->head->index_off);
-    store->end = (void *) ((uintptr_t) store->vma + store->vma_len);
+    store->head    = store->vma;
+    store->index   = (void *) ((uintptr_t) store->vma + store->head->index_off);
+    store->index_b = (void *) ((uintptr_t) store->vma + store->head->index_b_off);
+    store->data    = (void *) ((uintptr_t) store->vma + store->head->data_off);
+    store->data_b  = (void *) ((uintptr_t) store->vma + store->head->data_b_off);
+    store->end     = (void *) ((uintptr_t) store->vma + store->vma_len);
```

Comment:
While I understand the appeal of this type of code alignment, it does mean that whenever you add a new field, you will potentially have to realign all the field which is both annoying for people that don't subscribe to the glory of emacs and it pollutes the diff.

I don't mind too much as I have bought in to the glory of emacs but worth keeping in mind.

Reply:
Awrite, noted.

Reply:
Fixorzed.